docs/core/whats-new/dotnet-10/overview.md

```diff
@@ -0,0 +1,59 @@
+---
+title: What's new in .NET 10
+description: Learn about the new .NET features introduced in .NET 10 for the runtime, libraries, and SDK. Also find links to what's new in other areas, such as ASP.NET Core.
+titleSuffix: ""
+ms.date: 02/12/2025
```

Comment:
Might push the date out a bit.

Reply:
I was wondering this as well. I'd set it to the ship date for the preview before merging.